File: tests/example_data/cpp/function_used_twice.cpp
#include <cstdio>

inline void fun_0()
{
    std::putchar('{');
    std::putchar('%');
}

inline void fun_1()
{
    fun_0();
    std::putchar('\n');
    fun_0();
}

int main()
{
    fun_1();
    return 0;
}

File: tests/example_data/java/function_used_twice.java
public class Printer {
    public static void main(String[] args) {
        fun1();
    }

    static void fun0() {
        System.out.print('{');
        System.out.print('%');
    }

    static void fun1() {
        fun0();
        System.out.print('\n');
        fun0();
    }
}
